concourse/scripts/run_job.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash -eu
+set -o pipefail
+
+SCRIPT_DIR=$(cd "$(dirname "$0")" && pwd)
+PIPE='create-bosh-cloudfoundry'
+JOB=$1
+TEMP="/tmp/concourse-run-job.${RANDOM}"
+trap "rm -f $TEMP" ERR
+
+export TARGET_CONCOURSE=deployer
+# shellcheck disable=SC2091
+$("${SCRIPT_DIR}/environment.sh")
+"${SCRIPT_DIR}/fly_sync_and_login.sh"
+FLY="$FLY_CMD -t ${FLY_TARGET}"
+
+# 2 steps prevent fly reading empty input in case step 1 fails
```

Comment:
Yep, right you are. I hadn't realised that about chained pipes.

I'm not sure how important it is that we don't use the rendering scripts to prevent overwriting manual changes. Don't people normally make those changes in their local repo and upload them with `make dev pipelines` anyway?

I realise that it's a large change now, but I suspect it would also simplify the implementation if `pipelines-bosh-cloudfoundry.sh` took an additional environment variable that piped the manifest through `unbind_job.rb`. What do you think?